source/frontend/src/features/properties/map/shared/update/properties/UpdateProperties.tsx

```diff
@@ -92,18 +95,36 @@ export const UpdateProperties: React.FunctionComponent<
           {formikProps => (
             <FieldArray name="properties">
               {({ push, remove }) => (
-                <Row className="py-3 no-gutters">
-                  <Col>
-                    <MapSelectorContainer
-                      onSelectedProperty={(newProperty: IMapProperty) => {
-                        const formProperty = PropertyForm.fromMapProperty(newProperty);
-                        push(formProperty);
-                      }}
-                      existingProperties={formikProps.values.properties}
-                      onRemoveProperty={remove}
-                    />
-                  </Col>
-                </Row>
+                <>
```

Comment:
mmm, good to know that the update properties for the various files uses the same component under the hood. forgot about that.